src/metabase/lib/metadata/cached_provider.cljc

```diff
@@ -11,11 +11,25 @@
 
 #?(:clj (set! *warn-on-reflection* true))
 
+(def metadata-types
+  [:enum
+   :metadata/database :metadata/table :metadata/column
+   :metadata/card :metadata/metric :metadata/segment])
+
 (defn- get-in-cache [cache ks]
   (when-some [cached-value (get-in @cache ks)]
     (when-not (= cached-value ::nil)
       cached-value)))
 
+(mu/defn get-all-in-cache
```

Comment:
If this stuff is only used internally in this namespace please make it `^:private`. That way we don't have to figure out if it's used elsewhere if we want to change it in the future.

Reply:
made private